sources/web/datalab/static/settings.js

```diff
@@ -1,32 +1,42 @@
-function getSettingKeyAddress(setting) {
-  return window.location.protocol + "//" + window.location.host + "/_settings?key=" + setting;
-}
+define(['util'], (util) => {
+  function getSettingKeyAddress(setting) {
+    return window.location.protocol + "//" + window.location.host + "/_settings?key=" + setting;
+  }
 
-// Prepare the theme selector radio boxes
-lightThemeRadioOption = document.getElementById("lightThemeRadioOption")
-darkThemeRadioOption = document.getElementById("darkThemeRadioOption")
+  function init() {
```

Comment:
If we keep these console logs around, we should probably add a debug querystring or setting somewhere. For now, I think we should take these out